test/sendEmail.test.js

```diff
@@ -0,0 +1,18 @@
+const sendTestEmail = require('../mailing/sender.js');
+
```

Comment:
We'll want to add on the first line of this file:

```
require('dotenv').config();
```

so that environment variables are loaded before they are needed in the `sender.js` file. Usually we don't need to do this because we already import this file in `expressServer.js`, but since this test doesn't use `expressServer.js` we nede to import the dotenv manually here instead.

Reply:
Done , commit a77b200407833f06b57bd6bfb0cc4b2a29b164ed